### ISSUE TITLE
Add DFT contributions in tests

### ISSUE THREAD

**Issue by bast** [openrsp/openrsp#21]

**Comment by bingao** [comment]
I am not quite sure if we need to add some unit testing for DFT contributions for the current release, and if we have enough time to do it. Because I need to first run such a real calculation, for instance using LSDalton or Dalton, and fetch (un)perturbed matrices from the calculation.

For now, it is not straightforward for me to fetch such matrices from either LSDalton or Dalton. Therefore, I would suggest we postpone this issue after LSDalton+OpenRSP+DFT works.

**Comment by bingao** [comment]
Another reason is: I am more or less sure that the callback scheme should also work for DFT contributions. So that it might not be urgent to have such a test implemented.

**Comment by bast** [comment]
I agree that the unit test can and probably has to wait until the functionality is in the parent code so that we can generate the test data.

**Comment by ringholm** [comment]
Update from today: This will be added after @bast's XC work in LSDalton.

**Comment by ringholm** [comment]
that XC work now done, but remaining work for after release v1.0.0
